addons/mail/static/tests/discuss_app/jump_to_present.test.js

```diff
@@ -234,3 +235,62 @@ test("Post message when seeing old message should jump to present", async () =>
         after: [".o-mail-Message-content", { text: "Most Recent!" }], // should load around present
     });
 });
+
+test("jump to present triggers with at least 10 long messages seen", async () => {
+    // when messages are short, 3 x PRESENT_VIEWPORT_THRESHOLD is used, otherwise
+    // this is 10 x PRESENT_MESSAGE_THRESHOLD
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "channel",
+        name: "General",
+        channel_member_ids: [Command.create({ partner_id: serverState.partnerId })],
```

Comment:
Self member is automatically added to the channel and is actually irrelevant for this test: the jump to message banner should appear even for non-members. I'd suggest removing it; not a big deal, but this test is complicated, so it's better not to bloat it with irrelevant setup.